src/sdk/resources/policyActionTypes.ts

```diff
@@ -0,0 +1,9 @@
+import { IHttpClient } from '../http';
+import { PagedResource } from './resource';
+import { PolicyActionDescriptor } from '../models';
+
+export class PolicyActionTypesResource extends PagedResource<PolicyActionDescriptor> {
```

Comment:
Sorry, just noticed this. I think we just want this to extend `Resource` (get & list).

```suggestion
import { Resource } from './resource';
import { PolicyActionDescriptor } from '../models';

export class PolicyActionTypesResource extends Resource<PolicyActionDescriptor> {
```